pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -394,6 +412,11 @@ func loadBalancerServiceChanged(current, expected *corev1.Service) (bool, *corev
 		changed = true
 	}
 
+	if current.Annotations[awsLBAdditionalResourceTags] != expected.Annotations[awsLBAdditionalResourceTags] {
```

Comment:
nit: Add a comment explaining that we can safely reconcile these changes (See lines 396-401). Alternatively you could update the above comment.

Reply:
> nit: Add a comment explaining that we can safely reconcile these changes (See lines 396-401). Alternatively you could update the above comment.

Waiting on EP to see if we need to reconcile or only at creation time as proposed; https://github.com/openshift/enhancements/pull/706/commits/c0ccefa90fe8ac96607ac108adbbdf83874e3228#diff-9e7c9d80b69a46de2319af1610f54e1c0f04896901283d2e878bef4ecf915240R57